spec/components/contextual_breadcrumbs_spec.rb

```diff
@@ -97,6 +97,22 @@ def set_live_taxons(content_item)
     assert_select ".gem-c-breadcrumbs.gem-c-breadcrumbs--inverse"
   end
 
+  it "renders parent-based breadcrumbs if the content item is a corporate information page with a parent" do
+    content_item = example_document_for("corporate_information_page", "corporate_information_page_complaints")
+    render_component(content_item:)
+    assert_select "a", text: "Home"
+    assert_select "a", text: "Department of Health"
+    assert_select "a", text: "About us"
+  end
+
+  it "renders organisation breadcrumbs if the content item is a corporate information page with no parent but with a linked organisation" do
+    content_item = example_document_for("corporate_information_page", "best-practice-about-page")
+    render_component(content_item:)
+    assert_select "a", text: "Home"
+    assert_select "a", text: "Organisations"
+    assert_select "a", text: "Intellectual Property Office"
+  end
+
```

Comment:
Ah I hadn't clocked this. 
- Is it possible for a corporate information page to have no parent, but still have a link to an organisation? 
- Is it possible for a corporate information page to have a parent that is different to its linked organisation
- Is it possible for a corporate information page to be linked to more than one organisation?

Reply:
> Is it possible for a corporate information page to have no parent, but still have a link to an organisation?
Yes, that's the main problem this ticket resolves. E.g. https://www.gov.uk/api/content/government/organisations/government-actuarys-department/about

> Is it possible for a corporate information page to have a parent that is different to its linked organisation

TODO

> Is it possible for a corporate information page to be linked to more than one organisation?
Depending on what you mean by "possible".

I checked the schema and [this line](https://github.com/alphagov/publishing-api/blob/056cd207fad7f17aabb86a5f8ca0782258a55f96/content_schemas/dist/formats/corporate_information_page/frontend/schema.json#L131) just specified that it's frontend_links_with_base_path and [the definition of it](https://github.com/alphagov/publishing-api/blob/056cd207fad7f17aabb86a5f8ca0782258a55f96/content_schemas/dist/formats/corporate_information_page/frontend/schema.json#L492) is a kind of array that does not have any restriction on the number of entries. So in this sense, it's allowed by the schema.

On the other hand, I did a query in content store and it seems there are currently no records with more than one organisation linked. So in this sense, the answer is "no".

Reply:
Thanks for looking into this Mat - would you mind checking the code in Whitehall that creates corporate information pages - is it a model that has a relationship to an org? That would give us the confidence we need I think... everything else looks great.

Reply:
Yes, here's the line: https://github.com/alphagov/whitehall/blob/dedeaee2f486fd6c52cbfdc4a1077174ba4b01f8/app/models/corporate_information_page.rb#L11